.github/workflows/github-pages.yml

```diff
@@ -0,0 +1,19 @@
+name: Build and deploy to GitHub Pages
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+  push:
+    branches:
+      - develop
+
+jobs:
+  github-pages:
+    runs-on: ubuntu-16.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: helaili/jekyll-action@2.0.4
```

Comment:
Isn't there an official GH one?

Reply:
No, if you look at the [official docs](https://jekyllrb.com/docs/continuous-integration/github-actions/) you'll see that's the recommended action